stackslib/src/net/chat.rs

```diff
@@ -1999,9 +1984,29 @@ impl ConversationP2P {
         ) {
             Ok(Some(chunk)) => chunk,
             Ok(None) => {
-                // request for a stale chunk
+                // TODO: this is racey
```

Comment:
Do we have a ticket to address the raceyness or is it gonna get gixed in this PR?

Reply:
It's #5067